pandas/_libs/window/aggregations.pyx

```diff
@@ -1082,13 +1067,13 @@ def roll_min(ndarray[float64_t] values, ndarray[int64_t] start,
     return _roll_min_max(values, start, end, minp, is_max=0)
 
 
-cdef _roll_min_max(ndarray[numeric_t] values,
+cdef _roll_min_max(ndarray[float64_t] values,
```

Comment:
Interestingly we have type specialization built into this module but only ever call these functions with float64_t arrays. `-Wunused-function` does not warn about unused inline static functions so this went undetected

Reply:
Nice, yeah there's an issue to have rolling ops not always cast & return float64 but better to remove this for now